examples/manifests/kustomization.yaml

```diff
@@ -16,7 +16,10 @@ replicas:
 images:
 - name: cloud-event-proxy
   newName: quay.io/aneeshkp/cloud-event-proxy
+  newTag: latest
 - name: cloud-native-event-consumer
   newName: quay.io/aneeshkp/cloud-native-event-consumer
+  newTag: latest
 - name: cloud-native-event-producer
   newName: quay.io/aneeshkp/cloud-native-event-producer
```

Comment:
you don't need producer here anymore. The producer is cloud-event-proxy with hw-plugin enabled right ? You can delete the producer completely

Reply:
removed